frontend/src/scenes/pipeline/PipelineNodeNew.tsx

```diff
@@ -27,9 +27,11 @@ import { PipelineBackend } from './types'
 import { RenderApp } from './utils'
 
 const paramsToProps = ({
-    params: { stage, id },
+    params: { stage, id } = {},
+    searchParams: { kind } = {},
```

Comment:
what's the difference between `params` and `searchParams`?

they look like they could be collapsed?
maybe a comment for the future traveller?

Reply:
They're different things. `params` is matching on stuff like `/my_path/:source/:id`, it comes from Kea. `search_params` are browser `queryParams`